src/frontend/src/main.tsx

```diff
@@ -6,9 +6,7 @@ import './sass/common.scss';
 import { BrowserRouter } from 'react-router-dom';
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
-  <React.StrictMode>
-    <BrowserRouter>
-      <App />
-    </BrowserRouter>
-  </React.StrictMode>,
+  <BrowserRouter>
+    <App />
```

Comment:
This code temp was removed but will be added back later, the main reason is that React will render twice in the Dev(in production it only render once) environment at StrictMode and it causes a problem. I am working to find a solution after this.